### ISSUE TITLE
Anonymization of the Transaction Visualization Plots

### ISSUE THREAD

**Issue by Ly0n** [protontypes/LibreSelery#130]
Thanks @ZeipsCode we can now read the transaction per user plot.
At the moment we used the email address for the plot but we should stop that for the release.
The problem is that were not able to  translate the Github user email address into an user name when we include dependency scanning. That's why we add the user name to the transaction note at the beginning so that we can easily read it later on
```
@thisisdhaas:Thank you for your contribution to to PyGithub/PyGithub. We are using it at protontypes/openselery: Please give us a star if you like our project. Find out more about OpenSelery at:
https://github.com/protontypes/openselery.
```
In this way we can read the user name later on from the transaction history in an automated way and use that for the plot.

**Comment by Ly0n** [comment]
We now should have a transaction history that is long enough to cut off the transactions without the GitHub user at the beginning. In this way, new adaptors will have better plots.

**Comment by Ly0n** [comment]
Solved with a8a3b6104159522fee8b9f3ed78611546c9a6b35 
Thanks to @ZeipsCode
